app/controllers/credentials.controller.js

```diff
@@ -93,11 +93,14 @@ module.exports = {
     const { username, password } = _.get(req, 'body')
 
     if (!username) {
-      req.flash('genericError', `<h2>Please enter a valid username</h2>`)
+      req.flash('genericError', `Enter a username`)
     } else if (!password) {
-      req.flash('genericError', `<h2>Please enter a valid password</h2>`)
-    } else if (isPasswordLessThanTenChars(password)) {
-      req.flash('genericError', `<h2>Please enter a valid password</h2> ${isPasswordLessThanTenChars(password)}`)
+      req.flash('genericError', `Enter a password`)
+    } else {
+      const validationError = isPasswordLessThanTenChars(password)
```

Comment:
From the context of the code it looks like this will return a 'reason' string if invalid and null if valid? As this isn't an actual `Error` object potentially naming this `failedValidationMessage` or something more specific would clear this up.

Reply:
:+1: